src/app/sections/TelemetrySearchResults.svelte

```diff
@@ -201,23 +201,22 @@ li {
 
 <svelte:window bind:innerWidth={windowWidth} on:keydown={handleKeypress} />
 
-<Portal>
+<Portal customClasses={['telemetry-search-results-container']}>
```

Comment:
I think this can actually be done without adding `customClasses` as a prop. adding to `.telemetry-results` a `position:fixed; top: var(--increment);` seems to achieve the same effect with minimal change to `Portal`.

Reply:
Yep I am all for added extensibility only when needed and this might be solved in the child DOM node indeed. Should be quick regardless.